src/app/Caches/Create/ConfigurationBasic.tsx

```diff
@@ -0,0 +1,231 @@
+    const [isOpenNumberOwners, setIsOpenNumberOwners] = useState(false);
+    const [isOpenEncodingCache, setIsOpenEncodingCache] = useState(false);
+
+    useEffect(() => {
+        // Update the form when the state changes
+        handleFormConfigurationBasic(topology, selectedNumberOwners, selectedEncodingCache, isStatistics);
+    }, [topology, selectedNumberOwners, selectedEncodingCache, isStatistics]);
+
+
+    // Helper function for Number Owners Selection
+    const onToggleNumberOwners = () => {
+        setIsOpenNumberOwners(!isOpenNumberOwners);
+    };
+
+    const onSelectNumberOwners = (event, selection, isPlaceholder) => {
+        if (isPlaceholder) clearSelectionNumberOwners();
+        else {
+            setSelectedNumberOwners(selection);
+            setIsOpenNumberOwners(false);
+            console.log('selected:', selection);
+        }
+    };
+
+    const clearSelectionNumberOwners = () => {
+        setSelectedNumberOwners('');
+        setIsOpenNumberOwners(false);
+    };
+
+
+    // Helper function for Encoding Cache Selection
+    const onToggleEncodingCache = () => {
+        setIsOpenEncodingCache(!isOpenEncodingCache);
+    };
+
+    const onSelectEncodingCache = (event, selection, isPlaceholder) => {
+        if (isPlaceholder) clearSelectionEncodingCache();
+        else {
+            setSelectedEncodingCache(selection);
+            setIsOpenEncodingCache(false);
+            console.log('selected:', selection);
+        }
+    };
+
+    const clearSelectionEncodingCache = () => {
+        setSelectedEncodingCache('');
+        setIsOpenEncodingCache(false);
+    };
+
+    // Helper Function for Statistics checkbox
+    const handleIsStatistics = (checked, event) => {
+        setIsStatistics(event.target.checked);
+    }
+
+    // Form Topology
+    const formTopology = () => {
+        return (
+            <React.Fragment>
+                <FormGroup
+                    isInline
+                    isRequired
+                    fieldId="topology"
+                >
+                    <Radio
+                        name="radio"
+                        id="distributed"
+                        onChange={() => setTopology('distributed')}
+                        isChecked={topology === 'distributed'}
+                        label={
+                            <TextContent>
+                                <Text component={TextVariants.h3}>Distributed</Text>
+                            </TextContent>
+                        }
+                    />
+                    <Radio
+                        name="radio"
+                        id="replicated"
+                        onChange={() => setTopology('replicated')}
+                        isChecked={topology === 'replicated'}
+                        label={
+                            <TextContent>
+                                <Text component={TextVariants.h3}>Replicated</Text>
+                            </TextContent>
```

Comment:
@dipguptaredhat labels need to be used. there are some content constant with this values

Reply:
Labels are already in use, could you please elaborate.

Reply:
Replicated is not using the constant, the value is written in the code